data/json/overmap/overmap_mutable/crater.json

```
@@ -30,6 +30,13 @@
         "east": { "id": "crater_to_crater", "type": "available" },
         "south": { "id": "crater_to_crater", "type": "available" },
         "west": { "id": "crater_to_crater", "type": "available" }
+      },
+      "ground_cleanup": {
+        "overmap": "core_small",
```

Is this supposed to be?
```suggestion
        "overmap": "crater_core_small",
```